GaugeChart/go.mod

```diff
@@ -0,0 +1,30 @@
+module github.com/perses/plugins/gaugechart
+
+go 1.23.4
+
+require github.com/perses/perses v0.50.1
+
+require (
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cespare/xxhash/v2 v2.3.0 // indirect
+	github.com/go-jose/go-jose/v4 v4.0.4 // indirect
+	github.com/jpillora/backoff v1.0.0 // indirect
+	github.com/kr/text v0.2.0 // indirect
+	github.com/muhlemmer/gu v0.3.1 // indirect
+	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f // indirect
+	github.com/prometheus/client_golang v1.20.5 // indirect
+	github.com/prometheus/client_model v0.6.1 // indirect
+	github.com/prometheus/common v0.61.0 // indirect
+	github.com/prometheus/procfs v0.15.1 // indirect
+	github.com/zitadel/oidc/v3 v3.33.1 // indirect
+	github.com/zitadel/schema v1.3.0 // indirect
+	golang.org/x/crypto v0.31.0 // indirect
+	golang.org/x/net v0.32.0 // indirect
```

Comment:
I just merged a vulnerability upgrade, I think it should be:
```suggestion
	golang.org/x/net v0.33.0 // indirect
```

Reply:
done on all plugins